release/benchmark-worker-startup/test_single_configuration.py

```diff
@@ -0,0 +1,125 @@
+#!/usr/bin/env python3
+"""
+Helper file for benchmark_worker_startup.py. This file runs a particular test
+configuration.
+"""
+
+import argparse
+import ray
+import sys
+import time
+
+
+@ray.remote
+class Actor:
+    def run_code(self):
+        import torch  # noqa: F401
+
+
+@ray.remote
+def task():
+    import torch  # noqa: F401
+
+
+def main(
+    metrics_actor,
+    test_name: str,
+    num_runs: int,
+    num_tasks_or_actors_per_run: int,
+    num_cpus_in_cluster: int,
+    use_actors: bool,
+    with_gpu: bool,
+    with_runtime_env: bool,
+):
+
+    num_gpus = 0.0001 if with_gpu else 0
```

Comment:
huh i think this might not make sense since we usually doesn't share GPU? 
One way to simulate GPU is actually add custom GPU resource to only_head_node_1gpu_64cpu.yaml

Reply:
seems to me that it's suboptimal in either case:
* we use fractional GPUs on a 1 GPU machine (real workloads rarely use fractional workloads)
* we create 64 GPU resources on a 1 GPU machine (will Ray give each actor cuda:0? real workloads never do this either)

I can do the second one as you suggest. can you help me understand why this tradeoff is better?